src/PlaceholderComponent.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import reactStringReplace from 'react-string-replace';
+
+
+function PlaceholderTextComponent({ title }) {
```

Comment:
What happened to the design on the team design share? there were 3 props passed in to this component:

- text 
- regex
- cssClass

why was that changed? Looks like you embeded those props inside the title prop